java/requirements.txt

```diff
@@ -0,0 +1,3 @@
+black
```

Comment:
This file should conventionally be named requirements-dev.txt, I believe.

We might want to pin versions for each of these? And add a note to the README. (Sorry for the trouble here…)